addons/stock/models/stock_traceability.py

```diff
@@ -32,12 +32,17 @@ def _get_move_lines(self, move_lines):
                 ) - lines_seen
             # if MTS
             elif move_line:
-                lines = self.env['stock.move.line'].search([
+                domain = [
```

Comment:
I think it's better to create a function like '_prepare_domain' that you override in MRP than having a hasattr referencing an attribute added in another module